### PR TITLE
doc changes as per susan's review comment

### PR DESCRIPTION
**Pull request by ar-india**
Signed-off-by: ar-india <alokranjan2005@gmail.com>

### CODE REVIEW

**Review by rgcostea — APPROVED** [review]
lgtm

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py [hunk 1]
```diff
@@ -57,7 +57,7 @@
     required: False
     type: bool
     description:
-    - Check if the interfaces are offline before regenerating the WWNN (World Wide Node Name). Possible values:- 'true' 'false'.
+    - Check if the interfaces are offline before regenerating the WWNN. Possible values:- 'true' 'false'.
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py:60]: When True/False choices are rendered, it says Yes or No. Please remove all instances of: "Possible values:- 'true' 'false'"

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py [hunk 2]
```diff
@@ -69,14 +69,14 @@
     - present
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py:69]: Why is there no "absent" here? (pardon my lack of understanding how this particular API resource works)

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py [hunk 3]
```diff
@@ -69,14 +69,14 @@
...
     type: str
     description:
-    - Choice for fibre channel operation.
+    - The fibre channel operation.
   wwnn_base_str:
     required: False
     type: str
     description:
-    - Base World Wide Node Name(WWNN). Six bytes expressed in hex separated by colons. Example:- 'af:32:f1'.
+    - Base WWNN. Six bytes expressed in hex separated by colons. Example:- 'af:32:f1'.
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py:77]: Is it possible to write  as Example: "af:32:f1" instead?

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py [hunk 4]
```diff
@@ -75,15 +75,15 @@
         "volumes",
         "volume_collections"
       - NOTE:-
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py:77]: I see this convention ":- " used in some places, can we just use ": "?

Comment (datamattsson) [inline ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py:77]: This section renders very ugly: https://datamattsson.github.io/nimble-ansible-modules/modules/hpe_nimble_info_module.html#hpe-nimble-info-module

**Review by datamattsson — CHANGES_REQUESTED** [review]

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py (hunk 1 above)
Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py:60]: although the rendering converts it to yes and no but as per ansible guidelines: true false can be provided in multiple forms 
like 0,1 or yes or no, OR
true or false. Since our nimble spec has specified true or false and customers are used to it . I think it makes sense to leave it as it is. anyway there is no harm.

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py (hunk 2 above)
Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py:69]: because u cannot delete or remove FC. it has only one api. update.not even create . hence state is always present

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py (hunk 3 above)
Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py:77]: not possible.. ':' gives rendering error for ansible-doc command

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py (hunk 4 above)
Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py:77]: not possible.. ':' gives rendering error for ansible-doc command

Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py:77]: agreed but have no choice. :(

Comment (datamattsson) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py:77]: Since this text blob doesn't allow advanced formatting or lists, why try cram it in? Can't we just write the description in plain English?

Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py:77]: so r u suggesting to remove NOTE:- completely??

can u please paste the text how you want it to be ?

Comment (datamattsson) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py:77]:
```
When supplied, this argument will define the information to be collected. Possible values for this include "all" "minimum" "config" "access_control_records", "alarms", "application_servers", "application_categories", "arrays", "chap_users", "controllers", "disks", "fibre_channel_interfaces", "fibre_channel_configs", "fibre_channel_initiator_aliases", "fibre_channel_ports", "folders", "groups", "initiator_groups", "initiators", "master_key", "network_configs", "performance_policies", "pools", "protection_schedules", "protection_templates", "protocol_endpoints", "replication_partners", "shelves", "snapshots", "snapshot_collections", "software_versions", "user_groups", "user_policies", "users", "volumes", "volume_collections".


Each subset except "all", "minimum" and "config" supports four types of subset options. Subset "all" supports limit and detail as subset options. Subset "config" and "minimum" does not support any subset options.


See the example section for usage of the following subset options.
"fields" - A string representing which attributes to display for a given subset.
"limit" - An integer value which represents how many latest items to show for a given subset.
"detail" - A bool flag when set to true fetches everything for a given subset. Default is "True".
"query" - A key-value pair to query.
```

Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py:77]: this will look even bad.. 


```
See the example section for usage of the following subset options.
"fields" - A string representing which attributes to display for a given subset.
"limit" - An integer value which represents how many latest items to show for a given subset.
"detail" - A bool flag when set to true fetches everything for a given subset. Default is "True".
"query" - A key-value pair to query.
```

will be shown in one line and not in a separate line.

- "fields" - A string representing which attributes to display for a given subset.  ===> will give rendering error. 
we cannot use double quote after '-' immediately.

let's leave it as it is..
